### PR TITLE
Allow activesupport 5.1

### PR DESCRIPTION
**Pull request by jeznag**

### CODE REVIEW

File: lib/aemo/version.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # @author Joel Courtney <euphemize@gmail.com>
 module AEMO
   # aemo version
-  VERSION = '0.1.41'.freeze
+  VERSION = '0.1.42'
```
Comment (jufemaiz) [inline lib/aemo/version.rb]: Why the removal of freeze again @jeznag ?

File: aemo.gemspec [hunk 2]
```diff
@@ -23,11 +23,11 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 1.9.3'
 
   # Production Dependencies
-  s.add_dependency 'json',      '~> 2.1'
+  s.add_dependency 'json',      '>= 1.7.5'
```
Comment (jufemaiz) [inline aemo.gemspec]: Why >= 1.7.5 @jeznag ?

Comment (jeznag) [reply aemo.gemspec]: It didn't work with energylink otherwise

File: lib/aemo/version.rb (hunk 1 above)
Comment (jeznag) [reply lib/aemo/version.rb]: image

File: aemo.gemspec (hunk 2 above)
Comment (jufemaiz) [reply aemo.gemspec]: >< ok :)

File: lib/aemo/version.rb (hunk 1 above)
Comment (jufemaiz) [reply lib/aemo/version.rb]: 👌🏼

**Pull request merged**